target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,15 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_default "dsl" "DSL" "${board}:white:dsl" "0"
```

Comment:
Personally I don't like to add LEDs just to have them available in uci. But something similar was committed this week. Therefore I do **not** resist on removing this ucidef_set_led_default.